### PR TITLE
Add syntax highlighting

### PR DESCRIPTION
**Pull request by nixel2007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,95 +116,100 @@ applications you need to test the output of these applications. The methods
...
-			    System.out.println("first line");
-			    System.out.println("second line");
-			}
-		);
-		assertEquals(text, "first line\nsecond line");
-	}
+```java
+@Test
+void check_text_written_to_System_err(
+) throws Exception {
```
Comment (ashleyfrieze) [inline README.md:122]: Shall we remove the line break inside the `()`s while we're here?

File: README.md [hunk 2]
```diff
@@ -86,19 +90,21 @@ code each System property has the same value like before. Therefore you
...
-        }
-      );
-      
-      //Here the value of "some.property" is the same like before.
-      //E.g. it is not set.
+```java
+@Test
+void execute_code_that_manipulates_system_properties() {
+    restoreSystemProperties(
+    () -> {
```
Comment (ashleyfrieze) [inline README.md:97]: This section of the code might be better indented by one indent

File: README.md [hunk 3]
```diff
@@ -110,95 +116,100 @@ applications you need to test the output of these applications. The methods
...
-      assertNothingWrittenToSystemErr(
-        () -> {
-          System.err.println("some text");
-        }
-      );
+```java
+@Test
+void fails_because_something_is_written_to_System_err() {
+    assertNothingWrittenToSystemErr(
+    () -> {
```
Comment (ashleyfrieze) [inline README.md:172]: The indent has gone funny down here too

**Comment by stefanbirkner** [comment]
Thanks for the improvement. I merged your commit but without the changes that removed empty lines before new sections: 712ef87f101b3d4494a5938d44eaa5b8450ffd77

**Pull request closed without merging**